tests/integration/upsertGraph.js

```diff
@@ -1,11 +1,14 @@
 const raw = require('../../').raw;
 const expect = require('expect.js');
 const chai = require('chai');
+const chaiAsPromised = require('chai-as-promised');
 const Promise = require('bluebird');
 const transaction = require('../../').transaction;
 const ValidationError = require('../../').ValidationError;
 const mockKnexFactory = require('../../testUtils/mockKnex');
 
+chai.use(chaiAsPromised);
```

Comment:
This isn't used anywhere right? Why did you add it?

Reply:
yup, not needed anymore, was needed for promise rejection assertion